test/api/assignments_test.rb

```diff
@@ -116,7 +116,7 @@ def test_notify_team_members_about_submission
     Xapi.stub(:exists?, true) do
       solution = { "THE CODE" => "ruby/bob/code.rb" }
       post '/user/assignments', { key: bob.key, code: 'THE CODE', path: 'ruby/bob/code.rb',
-        'language' => 'ruby', 'problem' => 'bob', solution: solution }.to_json
+      'language' => 'ruby', 'problem' => 'bob', solution: solution }.to_json
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.